app/models/search_builder.rb

```diff
@@ -12,4 +15,16 @@ class SearchBuilder < Blacklight::SearchBuilder
   #   def add_custom_data_to_query(solr_parameters)
   #     solr_parameters[:custom] = blacklight_params[:user_value]
   #   end
+
+  self.default_processor_chain += [:min_match_for_boolean]
+
+  # Sets min match to 1 based on presence of boolean operators, overriding default set in Solr
+  def min_match_for_boolean(solr_parameters)
+    return unless blacklight_params &&
+                  blacklight_params[:q].present? &&
+                  blacklight_params[:q].respond_to?(:match) &&
+                  blacklight_params[:q].match(/\s(AND|OR|NOT)\s/)
```

Comment:
Didn't catch this earlier, but we could use `match?` here (and the line above) instead of `match` because we just want a true/false response and don't actually care about the matched values.

Reply:
Maybe something like:

```suggestion
    return unless search_state.query_param&.match?(/\s(AND|OR|NOT)\s/)
```

Reply:
applied suggestion